### PR TITLE
`Development`: Add exam exercise group edit/delete e2e test

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Currently the exam exercise group function includes tests, that create a group and an exercise within the group.

### Description

This PR adds exercise group deletion and editing tests. Additionally the group title is now properly checked.  

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2
#### Exam Mode Test
- [ ] Test

### CODE REVIEW

**Review by simon-hng — APPROVED** [review]
Code

**Review by akesfeden — APPROVED** [review]
Test additions look good and E2E tests pass on Bamboo, approved.

File: src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html [hunk 1]
```diff
@@ -16,18 +16,18 @@ <h4 class="mb-0">{{ 'artemisApp.examManagement.exerciseGroups' | artemisTranslat
 <div class="mt-3 text-end" id="number-groups" style="font-size: 0.85rem">
     {{ 'artemisApp.examManagement.numberOfExerciseGroups' | artemisTranslate }} {{ exerciseGroups ? exerciseGroups.length : 0 }}
 </div>
-<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3">
+<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3" id="{{ 'group-' + i }}">
```
Comment (heddendorp) [inline src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html:19]: Same as in the other PR, check with https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

File: src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html [hunk 2]
```diff
@@ -16,18 +16,18 @@ <h4 class="mb-0">{{ 'artemisApp.examManagement.exerciseGroups' | artemisTranslat
...
+            <a (click)="moveUp(i)" [class.disabled]="i === 0" class="border-0 p-0 me-1 bg-transparent">
                 <fa-icon [icon]="faAngleUp" style="font-size: 1.3rem"></fa-icon>
             </a>
             <a (click)="moveDown(i)" [class.disabled]="i >= exerciseGroups!.length - 1" class="border-0 p-0 bg-transparent">
                 <fa-icon [icon]="faAngleDown" style="font-size: 1.3rem"></fa-icon>
             </a>
         </div>
         <div class="d-flex flex-column justify-content-center">
-            <h5 class="font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
+            <h5 class="group-title font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
```
Comment (heddendorp) [inline src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html:30]: I think if these are for element selection data-cy is better suited as others might remove useless css classes

File: src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html (hunk 1 above)
Comment (TheZoker) [reply src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html:19]: I talked to @krusche about this and we decided that we are going to stick with the id selector for now.

**Pull request merged**